src/rpc/server.cpp

```diff
@@ -258,6 +258,22 @@ UniValue stop(const JSONRPCRequest& jsonRequest)
     return "Bitcoin server stopping";
 }
 
+UniValue uptime(const JSONRPCRequest& jsonRequest)
```

Comment:
Thank you very much for your feedback John.

I placed it on `src/rpc/server.php` because the command was information related to the server rather than a utility command (although it can certainly be considered an utility command). That is my only argument for using `server.cpp`. Maybe let's wait for other people's opinion on this?